lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -93,6 +93,19 @@ def email_alert_api_has_notification(notification)
         )
       end
 
+      def email_alert_api_unsubscribes_a_subscription(subscription_uuid)
+        stub_request(:post, unsubscribe_url)
+          .to_return(
+            status: 200,
+            body: { "deleted" => subscription_uuid }.to_json
+        )
+      end
+
+      def email_alert_api_does_not_unsubscribe_an_invalid_subscription(*)
+        stub_request(:post, unsubscribe_url)
+          .to_return(status: 404)
+      end
```

Comment:
Technically, this happens when a subscription can't be found for a UUID. Maybe the method name could be more like: `email_alert_api_has_no_subscription_for_uuid` or `email_alert_api_does_not_unsubscribe_a_missing_subscription` ? (I'm not very good at naming things)

Reply:
yeah this sounds better thanks. I was struggling here as well